test/core/send.test.js

```diff
@@ -223,6 +226,41 @@ describe('Core.Send', function () {
 				done();
 			}, 100);
 		});
+
+		it('should try image transport in IE11 if XHR fails.', function (done) {
+			const server = sinon.fakeServer.create(); // Catch AJAX requests
+
+			// Fake IE11
+			window.MSInputMethodContext = true;
+			document.documentMode = true;
+
+			(new Queue('requests')).replace([]);
+			Send.init();
+			const dummyImage = {
+				attachEvent: sinon.stub()
+			};
+			const i = window.Image;
+			window.Image = sinon.stub().returns(dummyImage);
+
+			server.respondWith([500, { "Content-Type": "plain/text", "Content-Length": 5 }, "NOT OK"]);
+			Send.addAndRun(request);
+			server.respond();
+			// Respond OK for the image
+			server.respondWith([200, { "Content-Type": "plain/text", "Content-Length": 2 }, "OK"]);
+			server.respond();
+
+			console.log(dummyImage);
```

Comment:
Was this meant to still be here?

Otherwise this is adding a chunk of complexity to the callback path but looks good and seems worth it for 3-5% of calls!

Reply:
good spot, i will remove the logging.

Yes, it's a bit busy on the callback area, but I also think worth it.